src/card.css

```
@@ -0,0 +1,77 @@
+.card {
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product {
+  display: flex;
+  min-height: 134px;
+  margin: 32px 19px 40px;
```

use paddings for your card
```suggestion
  margin-bottom: 40px;
```